lib/api/showtimes.ts

```diff
@@ -0,0 +1,32 @@
+import { getApiError } from '@/lib/utils/getApiError'
+import { apiClient } from './axios-config'
+import type { ShowtimeSearchParams, ShowtimeSearchResponse } from './types'
+
+export const showtimesApi = {
+  async searchShowtimes(
+    params?: ShowtimeSearchParams
+  ): Promise<ShowtimeSearchResponse> {
+    try {
+      // Build request body with only provided parameters
+      const requestBody: ShowtimeSearchParams = {}
+      if (params?.movie_id) {
+        requestBody.movie_id = params.movie_id
+      }
+      if (params?.cinema_id) {
+        requestBody.cinema_id = params.cinema_id
+      }
+      if (params?.date) {
+        requestBody.date = params.date
+      }
```

Comment:
I think you don't need to rebuild the `params` as the type is already the `ShowtimeSearchParams`